wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -44,7 +47,20 @@ <h1 class="w-header__title" id="header-title">
                 {% if description %}<div class="w-header__description">{{ description }}</div>{% endif %}
             </div>
             {% if search_url %}
-                <form class="col search-form" action="{% url search_url %}{% if query_parameters %}?{{ query_parameters }}{% endif %}" method="get" novalidate role="search">
+                <form
+                    class="col search-form"
+                    action="{% url search_url %}{% if query_parameters %}?{{ query_parameters }}{% endif %}"
+                    method="get"
+                    novalidate
+                    role="search"
+                    {% if not use_alt_search %}
+                        data-controller="w-swap"
+                        data-action="change->w-swap#searchLazy input->w-swap#searchLazy"
+                        data-w-swap-location-param="true"
```

Comment:
```suggestion
```

This may not be needed, pending discussion on method naming to differentiate between search URL behaviour and submitting a serialised form (GET) behaviour. 

See https://github.com/wagtail/wagtail/pull/9961#discussion_r1254384963